src/Tools/dotnet-monitor/Exceptions/ExceptionsOperation.cs

```diff
@@ -103,6 +103,12 @@ await using (Utf8JsonWriter writer = new(stream, new JsonWriterOptions() { Inden
                 writer.WriteString("moduleName", instance.ModuleName);
                 writer.WriteString("message", instance.Message);
 
+                if (IncludeActivityId(instance))
+                {
+                    writer.WriteString("activityId", instance.ActivityId);
```

Comment:
Let's write this as a sub-object so that way it can be wholesale ignored if need be or individual properties don't have too long of names:

```
"activity": {
  "id": "...",
  "idFormat": "..."
}
```